contribute/process/images/abiprocess.pu

```diff
@@ -0,0 +1,25 @@
+[*] --> NewABIChange: Firmware feature that requires ABI change
+
+NewABIChange --> RFC: Send RFC Pull Request to \nget driver and fw stakeholder feedback
+
+note right of RFC : RFC stage is intended\nto avoid wasted effort via \nearly engagement with the ABI users \n(driver developers)
+
+RFC --> FWImplementation: Ok proceed to implement full change
+
+RFC --> DriverImplementation: Start working on driver changes
+
+NewABIChange --> FWImplementation: Fast path implementation\nfor minor changes
+
+note left of FWImplementation : Fast path only when no impact to driver
+
+FWImplementation --> ABIClassification: Submit PR for classification
+
+ABIClassification -> DriverImplementation: Start working on driver changes
+
+note right of DriverImplementation: If 1) FW change is MAJOR or\n2) MINOR but modifies\nupstream topologies, then\nkernel change must\nbe merged at the same time
```

Comment:
what does "merged at the same time" mean and how does it help? You can click the two "merge" buttons within the same millisecond but it won't prevent users or distributions from mixing different versions?
"Upstream topologies" are topologies for platforms / configurations available in the kernel mainline (as opposed to only supported in sof-dev) or only those, whose names are used in one of machine drivers? E.g. is only sof-apl-pcm512x.tplg an upstream topology or all other sof-apl-*-pcm512x.tplg too?

Reply:
@lyakh I think we need to keep this a bit vague as we don't have infra to do cross-repo merges. This is not strictly related to the ABI, but rather our CI. Typically kernel side would have to be merged first, or otherwise CI will get broken (new topology files deployed to CI that the kernel does not know how to parse). As it's related to our CI, this consideration applies to all topologies we test in our CI.